seqr/views/apis/summary_data_api.py

```diff
@@ -172,6 +183,140 @@ def bulk_update_family_analysed_by(request):
     })
 
 
+def _load_aip_data(data, user):
+    category_map = data['metadata']['categories']
+    results = data['results']
+
+    family_id_map = dict(Individual.objects.filter(
+        family__project__in=get_internal_projects(), individual_id__in=results.keys(),
+    ).values_list('individual_id', 'family_id'))
+    missing_individuals = set(results.keys()) - set(family_id_map.keys())
+    if missing_individuals:
+        raise ErrorsWarningsException([f'Unable to find the following individuals: {", ".join(sorted(missing_individuals))}'])
```

Comment:
what's the reasoning for a combined error/warning exception?

Reply:
This is just a generic error type that we defined which has error handling middleware that properly handles this exception if it is raised and returns a well formatted httpresponse to the user with a 400 status code. Using it is just a way to throw an error that will be a 400 error message displayed to the user without having to reinvent the wheel for writing that logic